scripts/firedrake-install

```diff
@@ -593,7 +593,7 @@ def build_and_install_libspatialindex():
     if os.path.exists("libspatialindex"):
         libspatialindex_changed = git_update("libspatialindex")
     else:
-        git_clone("git+https://github.com/firedrakeproject/libspatialindex.git")
+        git_clone("git+https://github.com/firedrakeproject/libspatialindex.git@firedrake-dist")
```

Comment:
For an easy upgrade path, we should want to merge `firedrake-dist` into `master`.